components/core/DownloadDaemon.py

```diff
@@ -120,6 +122,10 @@ def worker(self):
                     download_id = get_id_from_gid(gid)
                     username = get_username_from_gid(gid)
                     send_status(download_id, completedLength, raw_size, username)
+                    file_name = path[-1]
+                    file_path=("/".join(path))
```

Comment:
Why not `os.path.join(*path)` ?

Reply:
that didn't provide `/` at starting and os.path.join(os.path.sep, *path) was being lengthy :sweat_smile: